ui/app/ducks/gas/gas.duck.js

```diff
@@ -233,29 +183,18 @@ export function fetchBasicGasEstimates() {
 async function fetchExternalBasicGasEstimates(dispatch) {
   const response = await queryEthGasStationBasic()
 
-  const {
-    safeLow: safeLowTimes10,
-    average: averageTimes10,
-    fast: fastTimes10,
-    fastest: fastestTimes10,
-    block_time: blockTime,
-    blockNum,
-  } = await response.json()
+  const { SafeGasPrice, ProposeGasPrice, FastGasPrice } = await response.json()
 
-  const [average, fast, fastest, safeLow] = [
-    averageTimes10,
-    fastTimes10,
-    fastestTimes10,
-    safeLowTimes10,
-  ].map((price) => new BigNumber(price).div(10).toNumber())
+  const [safeLow, average, fast] = [
+    SafeGasPrice,
+    ProposeGasPrice,
+    FastGasPrice,
+  ].map((price) => new BigNumber(price).toNumber())
```

Comment:
1. We're sure that converting these to `number` values is safe?
2. We should specify a `base` to the `BigNumber` constructor.

Reply:
> We're sure that converting these to number values is safe?

Safer than it was before this PR, at least, given that we are now relying on more reliable apis. These values should fall in the normal range of gas prices.

2. is addressed in 8ab057391